UPGRADING.INTERNALS

```diff
@@ -62,6 +62,10 @@ PHP 8.2 INTERNALS UPGRADE NOTES
 2. Build system changes
 ========================
 
+* Identifier names for namespaced functions generated from stub files through
```

Comment:
I fixed the conflict and slightly adjusted the wording. Feel free to change it should you find any issues with it.